python_modules/dagster/dagster/utils/error.py

```diff
@@ -2,11 +2,13 @@
 
 import traceback
 
-SerializableErrorInfo = namedtuple('SerializableErrorInfo', 'message stack')
+SerializableErrorInfo = namedtuple('SerializableErrorInfo', 'message stack cls_name')
```

Comment:
feel free to make this a "typed" named tuple (overriding __new__) if you are feeling frisky